lib/transmission-rss/feed.rb

```diff
@@ -0,0 +1,15 @@
+module TransmissionRSS
+  class Feed
```

Comment:
I would prefer `class Feed < Struct.new(:url, :regexp, :download_dir)` here instead of the `attr_reader` line.